src/components/InventoryTable/InventoryList.js

```diff
@@ -79,31 +20,20 @@ const InventoryList = React.forwardRef(
         <AccessDenied showReturnButton={false} />
       </div>
     ) : (
-      <ContextInventoryList
-        {...props}
-        activeFilters={activeFilters}
-        onRefreshData={onRefreshData}
-      />
+      <Grid gutter="sm" className="ins-inventory-list">
+        <GridItem span={12}>
+          <InventoryEntityTable {...props} onRefreshData={onRefreshData} />
```

Comment:
It seems that removing the redundant wrapper out of here has broken the fetch when sorting has changed. @bastilian was your intention to pass `onRefreshData` to `EntityTable` and refresh the data when sorting changes, from within the component?

Reply:
@johnsonm325 Do you have the current changes somewhere I can take a look at them?